doc/articles/faq.md

```diff
@@ -286,3 +286,87 @@ Yes, make sure to use the following project definition:
     <Reference Include="System.Runtime.Serialization" />
 </ItemGroup>
 ```
+
+## Does Uno offer a TreeView control?
+
+It's in the UWP API, but [not implemented yet](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tree-view)
+We have an open Github (issue.)[https://github.com/nventive/Uno/issues/3]
+
+## Is there a table of all UWP controls and their features compared to what's offered under Uno?
```

Comment:
'a table of all UWP controls' - @jeromelaban what's the status of the generated web pages? Are they going to see the light?

Reply:
They will, the doc site is being looked at currently.